bot/exts/evergreen/realpython.py

```diff
@@ -56,7 +56,7 @@ async def realpython(self, ctx: commands.Context, *, user_search: str) -> None:
         article_embed = Embed(
             title="Search results - Real Python",
             url=SEARCH_URL.format(user_search=quote_plus(user_search)),
-            description="Here are the top 5 results:",
+            description=f"Here are the top {len(articles)} results:",
```

Comment:
Special case for `len(articles) == 1`?

```suggestion
            description=f"Here are the top {len(articles)} results:" if len(articles) > 1 else 'Here is only one result:',
```

Reply:
It is over the 120 character limit for the code style, and I'm not sure how to break / wrap that inline if / else statement. Also I would use " instead of '

Would having a dedicated if / else just for the article_embed be too bulky?

Reply:
```py
        if len(articles) == 1:
            article_embed = Embed(
                title="Search result - Real Python",
                url=SEARCH_URL.format(user_search=quote_plus(user_search)),
                description=f"Here is the result:",
                color=Colours.orange,
            )
            
        article_embed = Embed(
            title="Search results - Real Python",
            url=SEARCH_URL.format(user_search=quote_plus(user_search)),
            description=f"Here are the top {len(articles)} results:",
            color=Colours.orange,
        )
```

Reply:
This was just a suggestion 👍 
You can also do something similar to the following code.

```py
        if len(articles) == 1:
            description="Here is the result:"
        else:
            description=f"Here are the top {len(articles)} results:"
            
        article_embed = Embed(
            title="Search results - Real Python",
            url=SEARCH_URL.format(user_search=quote_plus(user_search)),
            description=description,
            color=Colours.orange,
        )
```

Reply:
Yeah that actually looks way better and makes more sense. Are you able to make that in your suggested change? Then I can commit it directly from your comment.

Reply:
I will try :+1: